lib/node_modules/@stdlib/stats/base/snanmeanwd/test/test.ndarray.js

```diff
@@ -130,8 +126,7 @@ tape( 'the function supports a negative `stride` parameter', function test( t )
 		NaN   // 0
 	]);
 
-	N = floor( x.length / 2 );
-	v = snanmeanwd( N, x, -2, 6 );
+	v = snanmeanwd( 4, x, -2, 6 );
```

Comment:
```suggestion
	v = snanmeanwd( 5, x, -2, 8 );
```
please don't pass incorrect values into the tests, see how we want to traverse the array via the comments above